reports/sks444.md

```diff
@@ -0,0 +1,242 @@
+   - details: >
+     cEP-0020: Add Newcomer metrics and Gamification system
+ - 3:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/77f8802e1d573430ea7e1c56cbc240ed901ee427
+   - details: >
+     org/cron.py: Use dict.get method to r_json
+ - 4:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/791baab0f1fbafe035c8941bf2c36264798b65d4
+   - details: >
+     Add IGitt Cache mechanism
+ - 5:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/1b8058c044a805587ad3c240bcb3521121d25440
+   - details: >
+     requirements.txt: Use == for installing IGitt
+ - 6:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/00f3081ea17ba5a97e1b8032fc446af46f4ba4bf
+   - details: >
+     Serialize coala issues on GitHub and GitLab
+ - 7:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/10f622cc4902d8e61f95e0e872a26473c0646ba5
+   - details: >
+     Add teams to Contributor Model
+ - 8:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/b3caeceb1dba754b68027c3de0829b05f812f7d9
+   - details: >
+     Repository: Add filter_merge_requests method
+ - 9:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/bafb226be4c4834d127e16e38cbbfe661a2efcb7
+   - details: >
+     backend/db.sqlite3: Add old database files
+ - 10:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/29c2fc4c04d5ce57c1add937d6ea7d134def9953
+   - details: >
+     org/: Only pass valid usernames to methods
+ - 11:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/85372c2d5b7b9068c0f719bc0a288eb9307e9703
+   - details: >
+     org/cron.py: Fix contributors cron job
+ - 12:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/commit/d68a668482340e195ea678b5ce2d9b51e4616a85
+   - details: >
+     Serialize coala mrs on GitHub and GitLab
+ - 13:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/59/diffs?commit_id=0581b4c81c981eca81541b7f312beabf97393537
+   - details: >
+     org/: Use igitt_django models to save igitt data
+ - 14:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/60/diffs?commit_id=25e750d6f4f26e3dc7a10fabd8897a23e27d4332
+   - details: >
+     org/: Make an api to show active newcomers
+ - 15:
+   - repo: igitt-django
+   - link: https://gitlab.com/sks444/igitt-django/commit/824ced17fc6fbff379aef850ceccb5779d1f5a3e
+   - details: >
+     Switch to sqlite3
+ - 16:
+   - repo: webservices.coala.io
+   - link: https://github.com/sks444/webservices.coala.io/commit/9d263725092f2ab18fde84fb46dececc95d2a76d
+   - details: >
+     Generate a fake api similar to coala's webservices
+ - 17:
+   - repo: webservices
+   - link: https://gitlab.com/coala/landing/merge_requests/45/diffs?commit_id=993a68252f01c4ae7d78bfec22ba972b2fc5906e
```

Comment:
put all the unmerged stuff at the end of the list, and add 'WIP' on each of them, or 'POC', or 'Not reviewed' if review is pending.